backend/compact-connect/lambdas/python/common/cc_common/data_model/data_client.py

```
@@ -1293,6 +1386,278 @@ def encumber_license(self, adverse_action: AdverseActionData) -> None:
+                provider_status_items = self._generate_provider_encumbered_status_transaction_items_if_no_encumbrances(
+                    provider_user_records=provider_user_records,
+                    lifted_record=privilege_data,
+                )
+                transact_items.extend(provider_status_items)
+
+            # Execute the transaction
+            self.config.dynamodb_client.transact_write_items(TransactItems=transact_items)
+
+            logger.info('Successfully lifted privilege encumbrance')
+
+    def lift_license_encumbrance(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)